dashboard/app/models/user.rb

```diff
@@ -1567,6 +1568,12 @@ def sort_by_family_name?
     !!sort_by_family_name
   end
 
+  def get_show_progress_table_v2
+    return "default" if show_progress_table_v2.nil?
```

Comment:
You could just get rid of this method and return `show_progress_table_v2` directly from the API call. This would return `nil` for users who haven't set a preference, which I think would be entirely reasonable. This way is just fine too!

Reply:
I took your suggestion and simplified it

Reply:
I figure that it's a significant enough change that you might want a second glance